examples/plot_classifier_comp.py

```diff
@@ -1,8 +1,11 @@
-'''
-This script recreates the scikit-learn classifier comparison example found at http://scikit-learn.org/dev/auto_examples/plot_classifier_comparison.html.
-It has been modified to include an Earth based classifier.
-'''
+"""
+===============================
+Plotting sckit-learn classifiers comparison with Earth
+===============================
```

Comment:
number of = is not enough. I guess it breaks rendering. It should be the length of the number of characters in headline.